### PR TITLE
Add Intergas InComfort Lan2RF Gateway

### PR DESCRIPTION
**Pull request by zxdavb**
**Description:**
Adds the Intergas InComfort (known as InTouch in UK) Lan2RF gateway.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23736

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/incomfort.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  required: true
+  type: string
+username:
+  description: The username of the Lan2RF gateway
+  required: false
+  type: string
+password:
+  description: The password of the Lan2RF gateway
+  required: false
+  type: integer
```
Comment (klaasnicolaas) [inline source/_components/incomfort.markdown]: Are you sure this must be an integer?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/incomfort.markdown (hunk 1 above)
Comment (zxdavb) [reply source/_components/incomfort.markdown]: no - have pushed a fix

File: source/_components/incomfort.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+description: "Instructions on how to integrate an Intergas Lan2RF gateway with Home Assistant."
+date: 2019-03-03 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: incomfort.png
+ha_category:
+  - Water heater
+ha_release: 0.94
```
Comment (MartinHjelmare) [inline source/_components/incomfort.markdown]: This should make it into 0.93.

File: source/_components/incomfort.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ha_iot_class: Local Polling
+---
+
+The `incomfort` integration links Home Assistant with your Intergas Lan2RF gateway for controlling water_heater devices (the hub does not have to be in the same network as HA).
+
+The **Water Heater** device will report back the `state` and `temperature`. Other properties are available via the device's attributes.
+
+In future, other entities will be added to this integration (e.g. a **Sensor** for CV pressure, and a **Climate** device for room temperature).
```
Comment (MartinHjelmare) [inline source/_components/incomfort.markdown]:
```suggestion
In the future, other entities will be added to this integration (e.g. a **Sensor** for CV pressure, and a **Climate** device for room temperature).
```

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**